### PR TITLE
Make sure there is no inner fn decl

### PR DESCRIPTION
**Pull request by vicb**
/ref https://github.com/angular/angular/issues/13311

### CODE REVIEW

File: test/ws-webworker-context.ts [hunk 1]
```diff
@@ -8,6 +8,6 @@
 
 declare function importScripts(path: string): void;
 
-importScripts('/base/build/lib/zone.js');
-importScripts('/base/node_modules/systemjs/dist/system.src.js');
-importScripts('/base/build/test/zone_worker_entry_point.js');
+    importScripts('/base/build/lib/zone.js');
```
Comment (mhevery) [inline test/ws-webworker-context.ts]: this indentation does not seem right

File: test/mocha-patch.spec.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-declare function setup(fn: () => void): void;
-declare function teardown(fn: () => void): void;
-declare function suiteSetup(fn: () => void): void;
-declare function suiteTeardown(fn: () => void): void;
-declare function before(fn: () => void): void;
-declare function after(fn: () => void): void;
-//
-
-import {ifEnvSupports} from './test-util';
+    declare function test(description: string, testFn: () => void): void;
```
Comment (mhevery) [inline test/mocha-patch.spec.ts]: Indentation does not seem right here

File: test/main.ts [hunk 3]
```diff
@@ -6,28 +6,28 @@
...
 
-declare var __karma__: {
-  loaded: Function, start: Function
+declare var __karma__: {loaded: Function, start: Function
   error: Function
 };
 
-__karma__.loaded = function () { };
-(window as any).global = window;
+  __karma__.loaded = function() {};
```
Comment (mhevery) [inline test/main.ts]: Indentation here does not look right.

**Comment by mhevery** [comment]
Could we have the tslint run as part of the travis that way we don't go out of sync again?

File: test/ws-webworker-context.ts (hunk 1 above)
Comment (vicb) [inline test/ws-webworker-context.ts]: clang bug, we have to live with this for now :(

**Pull request merged**